Python-Function-LEN.md

```diff
@@ -0,0 +1,24 @@
+# Python len(x)
+
+`len()` is a built in function in Python 3. This method returns the length (the number of items) of an object. It takes one argument `x`. 
+
+## Arguments
+
+It takes one argument `x` - argument may be a sequence (such as a string, bytes, tuple, list, or range) or a collection (such as a dictionary, set, or frozen set).
+
+## Return Value
+
+This function returns the number of elements in the list
+
+## Code Sample
+
+```python
+list1 = [123, 'xyz', 'zara'] 
+str1 = 'basketball'
+print(len(list1)) # prints 3
+print(len(str1)) # prints 10
+```
+
+:rocket: [Run Code](https://repl.it/CUmt)
```

Comment:
Add more example with tuple and dictionaries. Also add comments as to why which value would be printed.